packages/lesswrong/lib/collections/arbitalTagContentRels/collection.ts

```diff
@@ -0,0 +1,35 @@
+import { createCollection } from '../../vulcan-lib';
+import { addUniversalFields, getDefaultResolvers, getDefaultMutations, } from '../../collectionUtils';
+import { userCanUseTags } from '../../betas';
+import { makeVoteable } from '../../make_voteable';
+import schema from './schema';
+
+export const ArbitalTagContentRels = createCollection({
+  collectionName: 'ArbitalTagContentRels',
+  typeName: 'ArbitalTagContentRel',
+  schema,
+  resolvers: getDefaultResolvers('ArbitalTagContentRels'),
+  mutations: getDefaultMutations('ArbitalTagContentRels', {
+    newCheck: (user: DbUser | null, document: DbArbitalTagContentRel | null) => {
+      return userCanUseTags(user);
+    },
+    editCheck: (user: DbUser | null, document: DbArbitalTagContentRel | null) => {
+      return userCanUseTags(user);
+    },
+    removeCheck: (user: DbUser | null, document: DbArbitalTagContentRel | null) => {
+      return false;
+    },
+  }),
```

Comment:
I suspect we don't actually want/need publicly-accessible mutations for this collection?  And even if we do we probably want to restrict new/edit to admins